Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            string userFirstName = GetStringInput(message: "Enter your first name: ");
+            string userSurname = GetStringInput(message: "Enter your surname: ");
+            int userAge = GetIntInput(message: "Enter your age: ");
+            float userHeight = GetFloatInput(message: "Enter your height in cm: ");
+            float userWeight = GetFloatInput(message: "Enter your weight in kg: ");
+            float userBMI = CalculateBmi(userWeight, userHeight);
+            Console.WriteLine(userFirstName + " " + userSurname + " is " + userAge + " years old, his weight is " + userWeight + " kg and his height is " + userHeight + " cm.");
+            Console.WriteLine("BMI = " + userBMI);
+        }
+
+        public static float CalculateBmi(float weight, float height)
+        {
+            if (weight <= 0 && height <=0)
+            {
+                Console.WriteLine("Failed calculating BMI. Reason:");
+                Console.WriteLine("Weight cannot be equal or less than zero, but was " + weight);
+                Console.WriteLine("Height cannot be equal or less than zero, but was " + height);
+            }
+            else if (weight <=0)
+            {
+                Console.WriteLine("Failed calculating BMI. Reason:");
+                Console.WriteLine("Weight cannot be equal or less than zero, but was " + weight);
+            }
+            else if (height <=0)
+            {
+                Console.WriteLine("Failed calculating BMI. Reason:");
+                Console.WriteLine("Height cannot be equal or less than zero, but was " + height);
+            }  
+                
+            float bmi = weight / (height * height);
+            return bmi;
+        }
+
+        public static float GetFloatInput(string message)
+        {
+            Console.Write(message);
+            var floatInput = Console.ReadLine();
+            if (string.IsNullOrEmpty(floatInput))
+            {
+                Console.WriteLine(floatInput + " is not a valid number.");
+                return 0;
+            }
+            else if (!float.TryParse(floatInput, out float number))
+            {
+                Console.WriteLine(floatInput + " is not a valid number.");
+                return -1;
+            }
+            else return float.Parse(floatInput, CultureInfo.InvariantCulture.NumberFormat);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'else' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4886045)